src/components/ArticleCard.jsx

```diff
@@ -1,5 +1,26 @@
-import React from "react";
+import { Image } from "react-bootstrap";
 
-export default function ArticleCard() {
-  return <div></div>;
+export default function ArticleCard({
```

Comment:
Nice use of bootstrap throughout for styling 🎨

Reply:
Thanks :)